src/main/java/io/vertx/codegen/MethodKind.java

```diff
@@ -16,9 +16,14 @@
   HANDLER,
 
   /**
-   * A method has a future kind when it has at least one parameter and its last parameter
+   * The method has a callback kind when it has at least one parameter and its last parameter
    * is a parameterized handler type having a <i>async_result</i> parameterized type argument.
    */
+  CALLBACK,
```

Comment:
This is confusing with `HANDLER`, because this definition of `HANDLER` is still a callback, in a certain sense :) 

What about naming this one `ASYNC_HANDLER`?

Reply:
I believe it is not anymore, because handler are for event and can be called multiple times. In Vert.x most (all but one :-) ) handler have been converted to callback + future

Reply:
I see what you mean, but for somebody that doesn't know the vert.x details like the one you sad, both `Handler<E>` and `Handler<AsyncResult<E>>` looks like callbacks, hence the term IMO is too generic. While `ASYNC_HANDLER` clearly identify `Handler<AsyncResult<E>>`